_sass/objects/_newsletter.scss

```diff
@@ -1,5 +1,7 @@
 .newsletter {
-  background-image: $white;
+  background-image: url('/img/PaysagePanda.png');
```

Comment:
this image is not optimized: could be ~50KB instead of ~200KB. also it needs to be resized properly because I don't think we need a width of ~1400px